Doc-804: Document new skin settings (#1931)
tinymce/tinymce-docs: release-notes/release-notes58.md

- Added a new `tinycomments_resolve` option for adding and configuring resolve conversation functionality for Comments in callback mode. For details, see: [Configuring comments callbacks - `tinycomments_resolve`]({{site.baseurl}}/advanced/configuring-comments-callbacks/#tinycomments_resolve).
- Added a new `tinycomments_can_resolve` option for adding and configuring resolve conversation functionality for Comments in embedded mode. For details, see: [Comments plugin - `tinycomments_can_resolve`]({{site.baseurl}}/plugins/premium/comments/#tinycomments_can_resolve).

### Tiny Drive 1.4.0

The {{site.productname}} 5.8 release includes an accompanying release of **Tiny Drive**.

**Tiny Drive** 1.4.0 provides the following new features:

> - Added new [tinydrive_skin]({{ site.baseurl }}/tinydrive/configuration/#tinydrive_skin) setting for configuring the skin to use in plugin mode.

Reviewer: ```suggestion
- Added a new [`tinydrive_skin`]({{ site.baseurl }}/tinydrive/configuration/#tinydrive_skin) option for applying a skin to the {{site.productname}} {{site.cloudfilemanager}} plugin.
```

> - Added new [skin]({{ site.baseurl }}/tinydrive/tinydrive-api/standalone/#skin) setting for configuring the skin to use in standalone mode mode.

Reviewer: ```suggestion
- Added a new [`skin`]({{ site.baseurl }}/tinydrive/tinydrive-api/standalone/#skin) option for applying a skin to standalone instances of {{site.cloudfilemanager}}.
```

> - Added new 'oxide-dark' skin.

Reviewer: Is this for standalone mode? or both?

Author: Both, should we mention that?

Reviewer: cool. yes please :)

> - Added new PDF viewer to enable previewing of PDF files.

Reviewer: ```suggestion
- Added a new PDF viewer to add previews for PDF files.
```

> - Added new 'oxide-dark' skin for standalone and plugin mode.

Reviewer 2: ```suggestion
- Added a new `oxide-dark` skin for standalone and plugin mode.
```

**Tiny Drive** 1.4.0 provides the following bug fixes:

> - Fixed a bug where images with exif orientation would get incorrectly oriented.

Reviewer: ```suggestion
- Fixed a bug where images with `exif` orientation would be incorrectly oriented.
```

> - Fixed a bug where specifying 'audio' for the `filetype` setting would not list the files in grid view.

Reviewer: ```suggestion
- Fixed a bug where specifying 'audio' for the `filetype` setting would not list the files in the grid view.
```
